component/vshn_postgres.jsonnet

```
@@ -776,6 +778,65 @@ local prometheusRule = {
                     'for': '120m',
                     labels: {
                       severity: 'critical',
+                      syn_team: 'schedar',
+                    },
+                  },
+                ],
+              },
+              // new
+              {
+                name: 'postgresql-replication',
+                rules: [
+                  {
+                    alert: 'PostgreSQLReplicationCritical',
```

```suggestion
                    alert: 'PostgreSQLPodReplicasCritical',
```

I feel that makes it clearer what exactly is not working.